autoload/lsp/diag.vim

```diff
@@ -27,6 +27,11 @@ def DiagsRefreshSigns(lspserver: dict<any>, bnr: number)
   # Remove all the existing diagnostic signs
```

Comment:
I noticed that when loading multiple buffers, there are error messages about trying to add a text property to an unloaded buffer.  At the beginning of this function, you need to add

   bnr->bufload()

to fix these issues.

Reply:
added in https://github.com/yegappan/lsp/pull/187/commits/6a7464d2dfa68fa5e4525e876712bcbed5f0300b